codegen/JsonUtils/src/test/resources/Q6.json

```diff
@@ -0,0 +1,49 @@
+{
+  "RelationProcessFunction": {
+    "name": "lineitem",
+    "relation": "lineitem",
+    "this_key": [
+      "orderkey",
+      "l_linenumber"
+    ],
+    "next_key": null,
+    "child_nodes": 0,
+    "is_Root": true,
+    "is_Last": true,
+    "rename_attribute": null,
+    "select_conditions": {
```

Comment:
The select condition is missing four, should also have l_shipdate < 1995-01-01 and l_discount >= 0.05 and l_discount <= 0.07 and l_quantity < 24

Reply:
Ops. BTW, we should also add logical operators like `AND` and `OR` capture how the select conditions are applied.

Reply:
You are right, then I think a recursive structure like expression is also needed here.